tests/plugincode/test_plugincode.py

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# http://nexb.com and https://github.com/nexB/scancode-toolkit/
+# The ScanCode software is licensed under the Apache License version 2.0.
+# Data generated with ScanCode require an acknowledgment.
+# ScanCode is a trademark of nexB Inc.
+#
+# You may not use this software except in compliance with the License.
+# You may obtain a copy of the License at: http://apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+#
+# When you publish or redistribute any data created with ScanCode or any ScanCode
+# derivative work, you must accompany this data with the following acknowledgment:
+#
+#  Generated with ScanCode and provided on an "AS IS" BASIS, WITHOUT WARRANTIES
+#  OR CONDITIONS OF ANY KIND, either express or implied. No content created from
+#  ScanCode should be considered or used as legal advice. Consult an Attorney
+#  for any legal advice.
+#  ScanCode is a free software code scanning tool from nexB Inc. and others.
+#  Visit https://github.com/nexB/scancode-toolkit/ for support and download.
+
+import plugincode
+from plugincode import location_provider
+from plugincode import output_filter
+from plugincode import output
+from plugincode import post_scan
+from plugincode import pre_scan
+from plugincode import scan
+
+import pytest
+pytestmark = pytest.mark.scanpy3  # NOQA
+
+def test_dummy():
+    pass
```

Comment:
To make it more explicit that we are only testing the import could instead move them to the test function?
Also you are missing basic imports for future and absolute imports

```
def test_plugincode_can_be_imported():
    import plugincode
    from plugincode import location_provider
    from plugincode import output_filter
    from plugincode import output
    from plugincode import post_scan
    from plugincode import pre_scan
    from plugincode import scan
```